metaseq/data/jsonl_dataset.py

```diff
@@ -80,31 +80,15 @@ def __getitem__(self, idx):
         # For instance, for a data_subshard_count of 3 and epoch number of 1,
         # subshard_idx goes like 0, 3, 6, 9 ...
```

Comment:
let's leave a link to this PR in the comments to help future dev's find  the _why_

Reply:
Updated